django_ilmoitin/templates/admin/preview_template.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'admin/change_form.html' %}
+
+{% block extrastyle %}
+    {{ block.super }}
+
+    <link rel="stylesheet" href="/static/admin/css/modal.css">
+{% endblock %}
+
+{% block extrahead %}
+    {{ block.super }}
+
+    <script src="https://unpkg.com/micromodal/dist/micromodal.min.js"></script>
+    <script src="/static/admin/js/preview.js"></script>
+{% endblock %}
+
+{% block submit_buttons_bottom %}
+    {{ block.super }}
+    <div class="submit-row">
+            <input type="button" value="Preview" name="_preview" onclick="showPreviewModal()"
```

Comment:
would be slightly more user friendly IMO, if the row with the "Preview" button came before the row with the "Save" buttons and if the "Preview" button was left-aligned 🤔